server/server.test.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const serverRoutes = require("./routes/rpaFramework"); 
+const request = require("supertest");
+const DBManager = require('./utils/TestingUtils/MongoDbManager');
+const DbTestingMatchers = require('./utils/TestingUtils/DbTestingMatchers');
```

Comment:
Do we have different naming schemes for Database = DB on purpose here? Otherwise let's always use "DB"